schutzbot/Jenkinsfile

```diff
@@ -675,36 +675,6 @@ pipeline {
                         }
                     }
                 }
-                stage('EL8 aarch64 Image') {
```

Comment:
Should this be part of this patch?

Reply:
Nope, squashed into the previous commit.